CMakeLists.txt

```diff
@@ -286,7 +286,9 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang"
     target_compile_options(ds2 PRIVATE -Wsuggest-override)
   endif ()
   if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    target_compile_options(ds2 PRIVATE -Winconsistent-missing-override)
+    if (NOT "${OS_NAME}" MATCHES "FreeBSD")
```

Comment:
Instead of checking for the OS, you should check for the compiler version. We already do something similar for checking the gcc version, see line 285.